spacy/cli/convert.py

```diff
@@ -110,6 +111,15 @@ def convert(
     input_path = Path(input_path)
     if not msg:
         msg = Printer(no_print=silent)
+
+    # Throw error for renamed language codes in v4
+    if lang in RENAMED_LANGUAGE_CODES:
+        msg.fail(
+            title="Renamed language code",
+            text=f"Language code '{lang}' was replaced with '{RENAMED_LANGUAGE_CODES[lang]}' in v4. Please change your current defined language code from '{lang}' to '{RENAMED_LANGUAGE_CODES[lang]}'.",
+            exits=1,
+        )
```

Comment:
Since this is repeated, let's move it to a method in `spacy.cli._util`. It could be a private method like `_handle_renamed_language_codes` or something clunky.

We can remove `lang` from `convert` in a follow-up PR.

For the error message, maybe:

```
            text=f"Language code '{lang}' was replaced with '{RENAMED_LANGUAGE_CODES[lang]}' in spaCy v4. Update the language code from '{lang}' to '{RENAMED_LANGUAGE_CODES[lang]}'.",
```

If all of these are `--lang lg`, then it could make sense to provide the full option in the error message.

Reply:
Created `_handle_renamed_language_codes` in `cli._util.py`, replaced all code blocks, and adjusted the error message.
I think, I don't fully understand what you mean by:
> If all of these are --lang lg, then it could make sense to provide the full option in the error message.